modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/TcsControllerEpics.scala

```diff
@@ -306,15 +341,20 @@ object TcsControllerEpics extends TcsController {
     _ <- setProbeTrackingConfig(TcsEpics.instance.oiwfsProbeGuideCmd, c.oiwfs.self)
   } yield TrySeq(())
 
+  // Special case: if source is the sky and the instrument is at the bottom port (port 1), the science fold must be parked.
   def setScienceFoldConfig(sfPos: ScienceFoldPosition): SeqAction[Unit] = sfPos match {
     case ScienceFoldPosition.Parked => TcsEpics.instance.scienceFoldParkCmd.mark
-    case p: ScienceFoldPosition.Position => TcsEpics.instance.scienceFoldPosCmd.setScfold(encode(p))
+    case p@ScienceFoldPosition.Position(LightSource.Sky, inst) => getInstPort(inst).flatMap(port =>
+      if (port == 1) TcsEpics.instance.scienceFoldParkCmd.mark.some
```

Comment:
What's the port here? ISS port? Perhaps the `1` should be made a more meaningful constant